indigo_api/models/works.py

```diff
@@ -34,12 +34,14 @@ class WorkManager(models.Manager):
 
     def get_queryset(self):
         # defer expensive or unnecessary fields
-        return super(WorkManager, self) \
-            .get_queryset() \
+        return super().get_queryset() \
             .select_related('updated_by_user', 'created_by_user', 'country',
                             'country__country', 'locality', 'publication_document') \
             .prefetch_related('commencements')
```

Comment:
commencements are potentially huge, I'm not sure we ALWAYS want to get them whenever we load a work. Can this prefetch be moved to where it's needed?

Reply:
I've opened https://github.com/laws-africa/indigo/pull/2114 for this

Reply:
sorry, I don't know how I misread this as a new addition 🤦